frontend/packages/operator-lifecycle-manager/src/components/catalog-source.tsx

```diff
@@ -126,19 +126,19 @@ export const CatalogSourceDetails: React.FC<CatalogSourceDetailsProps> = ({
       <div className="co-m-pane__body">
         <div className="col-xs-4">
           <dl className="co-m-pane__details">
-            <dt>{t('catalog-source~Name')}</dt>
+            <dt>{t('olm~Name')}</dt>
```

Comment:
CatalogSource Details heading is not translated
<img width="609" alt="Screen Shot 2020-12-15 at 6 56 07 PM" src="https://user-images.githubusercontent.com/12692381/102206575-d113f880-3f07-11eb-92f7-d9ac0b901dbe.png">

Reply:
PackageManifest details heading is not translated
<img width="624" alt="Screen Shot 2020-12-15 at 6 55 09 PM" src="https://user-images.githubusercontent.com/12692381/102206823-2819cd80-3f08-11eb-9b31-e03b2632e8e3.png">

Reply:
These are actually using the DefaultDetailsPage component, which is outside the scope of this PR.